src/attitude-utils.cpp

```diff
@@ -425,6 +425,47 @@ Vec3 Vec3::operator*(const Mat3 &other) const {
 ///// VECTOR UTILITY FUNCTIONS ////
 ///////////////////////////////////
 
+/**
+* Finds the midpoint between two different vectors
+*
+* @param vec1 The first vector
+* @param vec2 The second vector
+*
+* @return The mid point vector
+*
```

Comment:
As a convention, the there should be no space between the last content and the `*/` character (remove this space everywhere)